### DISCUSSION TITLE
Is there a way to make certain additional CSS that is added only apply to specific pages?

### DISCUSSION THREAD

**Discussion by Fatih20** [squidfunk/mkdocs-material#2893] [Q&A] [answered]
I wanted elements in my home sections to have different specifications to elements in my installation section. Is there a way to do as the title says? I know that I could just put all elements in the page to one class and all elements in the installation page to another, and then add any page-specific specification to those classes, but is there a more elegant way to do this?

**Comment by squidfunk** [comment, marked as answer]
Yes, you can use HTML in Markdown:

``` html
<link rel="stylesheet" href="link/to/stylesheet" />
<style>
  /* Inline styles */
</style>
```

Note that the stylesheet link probably has to be absolute, or it won't work correctly.

Alternatively, you can use [theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme), as Material for MkDocs does itself for the homepage, and provide a separate template for each site, which adds the relevant stylesheets.

https://github.com/squidfunk/mkdocs-material/blob/619d47f562ead9a1c988068e3b2a62dcbdbd8483/src/overrides/main.html#L25-L33

**Reply by Fatih20** [reply]
Is the `style` just placeholder and doesn't need to be filled with anything?

**Reply by squidfunk** [reply]
You could _either_ use `link` _or_ `style`, depending on whether you want to define things inline or external. The example was meant to illustrate that both ways are possible.

**Reply by Fatih20** [reply]
> You could _either_ use `link` _or_ `style`, depending on whether you want to define things inline or external. The example was meant to illustrate that both ways are possible.

Ah, alright. Thanks!
